doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_cli.rst

```diff
@@ -7,6 +7,12 @@ Light Lightness Control Client
    :local:
    :depth: 2
 
+.. note::
```

Comment:
Shouldn't this be added to the light_ctrl_srv.rst also?

Reply:
Should it? As far as I remember there is no API changes in the server, only changes to internal implementation details. Basically, if there is no difference in the 2.5 and 2.6 docs (which I believe is the case for light_ctrl_srv?) there is no point in having this note about looking up stuff in the old docs.